content/tracing/advanced_usage/_index.md

```diff
@@ -2015,13 +2015,19 @@ var tracer = Datadog.Trace.Tracer.Create(isDebugEnabled: true);
 
 Debug mode is disabled by default. To enable it, set the environment variable `DD_TRACE_DEBUG=true`.
 
+*An important note*: If you use code auto-instrumentation (the recommended approach) please be aware that the
+instrumenting code is executed before any user code. So the environment variables below must be set at the server
+level and be available to the PHP runtime before any user code is executed. So, for example, `putenv()` and `.env`
+files would not work.
+
 ```php
-putenv('DD_TRACE_DEBUG=true');
+DD_TRACE_DEBUG=true
```

Comment:
For the examples, it might be good to include the main suggested ways to add the environment variables depending on your server. Since we only support Apache, FPM & CLI server, we could have a short blurb and example of each. :)

### Apache

Set using [`SetEnv`](https://httpd.apache.org/docs/2.4/mod/mod_env.html#setenv) from the server config, virtual host, directory, or **.htaccess** file.

```
SetEnv DD_TRACE_DEBUG true
```

### nginx

Set using [`fastcgi_param`](http://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#fastcgi_param) from the `http`, `server`, or `location` contexts.

```
fastcgi_param DD_TRACE_DEBUG true;
```

### PHP CLI server

Set in the command line to start the server.

```
DD_TRACE_DEBUG=true php -S localhost:8888
```

Reply:
Good idea @SammyK , will add it :)